koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -746,11 +752,18 @@ def createContainer(self, src=None, target_info=None, arches=None,
                         scratch=None, isolated=None, yum_repourls=None,
                         branch=None, koji_parent_build=None, release=None,
                         flatpak=False, signing_intent=None, compose_ids=None,
+                        skip_build=None,  # BW compatibility
                         dependency_replacements=None, operator_csv_modifications_url=None,
                         userdata=None):
         if not yum_repourls:
             yum_repourls = []
 
+        if skip_build is not None:
+            if skip_build:
+                raise koji.BuildError("'skip_build' functionality has been removed")
+            else:
+                self.logger.warning("deprecated option 'skip_build' in build params")
```

Comment:
Could this be a user warning? Or does koji not support that?

Reply:
nope, user warning can go only form OSBS itself. It would need a lot of magic to add it into user warning and it's not worth.